lib/skate/settings/db/test_group.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Skate.Settings.Db.TestGroup do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Skate.Settings.Db.TestGroupUser, as: DbTestGroupUser
+
+  @type t() :: %__MODULE__{}
+
+  schema "test_groups" do
+    field(:name, :string)
+    timestamps()
+
+    has_many(:test_group_users, DbTestGroupUser, on_replace: :delete_if_exists)
+    has_many(:users, through: [:test_group_users, :user])
```

Comment:
Very cool - I've never seen the `through` association before!

Reply:
I haven't used them much either. They have some limitations (Ecto is good at reading them but for writing them you have to implement much more of the logic yourself, as I did here), but definitely a good tool to have!